packages/trader/src/Modules/SmartChartBeta/Components/LaunchModal/__tests__/launch-modal.spec.tsx

```diff
@@ -0,0 +1,112 @@
+import * as React from 'react';
+import { mockStore } from '@deriv/stores';
+import { render, screen } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import TraderProviders from '../../../../../trader-providers';
+import { LocalStore, TURBOS, WS } from '@deriv/shared';
+import Trade from 'Modules/Trading';
+import { useTraderStore } from 'Stores/useTraderStores';
+import { BrowserRouter } from 'react-router-dom';
+import { ActiveSymbols } from '@deriv/api-types';
+import ReactDOM from 'react-dom';
+
+const default_mocked_store = mockStore({
+    client: { is_logged_in: true },
+    common: { network_status: { class: 'online' } },
+    ui: { is_mobile: false, is_dark_mode_on: false, notification_messages_ui: 'address' },
```

Comment:
These are default values of `is_mobile` and `is_dark_mode_on`. If you are not going to change them in test cases, then no need in adding default values one more time. You can always check default value for tests in _packages/stores/src/mockStore.ts_
But anyway, I don't think you need all these mocks because of the comment above (https://github.com/binary-com/deriv-app/pull/11196/files#r1395388829)